jenkins/kernel-defconfig-creator.sh

```diff
@@ -165,6 +165,15 @@ if [ ${TREE_NAME} = "lsk" ] || [ ${TREE_NAME} = "anders" ]; then
   fi
 fi
 
+# Ard specific tree and branch defconfigs.
```

Comment:
I'm going to assume these are only wanted for ARM arch? How they are right now, they will be added to every architecture and will presumably fail on all the others.